config/samples/azure_v1alpha2_mysqluser.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: azure.microsoft.com/v1alpha2
+kind: MySQLUser
+metadata:
+  name: mysqluser-sample
+spec:
+  server: mysqlserver-sample
+  dbName: mysqldatabase-sample
+  resourceGroup: resourcegroup-azure-operators
```

Comment:
super duper mega minor: IMO put `resourceGroup` above `server` as it's the "parent" in the ownership hierarchy and it feels more natural to read that top to bottom (parent to child to childs-child and so on)

Reply:
Same applies to above sample too

Reply:
Yup yup, definitely agree about the hierarchy here.